lib/msf/core/module/ui/message.rb

```diff
@@ -14,9 +14,8 @@ def print_good(msg='')
 
   def print_prefix
     prefix = ''
-    if (datastore['TimestampOutput'] =~ /^(t|y|1)/i) || (
-      framework && framework.datastore['TimestampOutput'] =~ /^(t|y|1)/i
-    )
+    if datastore['TimestampOutput'] ||
```

Comment:
Could I just register the global option like any other instead, rather than making it magic? Or if we can set a default value to a global, maybe we could use type inference to make sure it doesn't become polymorphic.